### ISSUE TITLE
Suggestion: Increase number of measures

### ISSUE THREAD

**Issue by RubyTheDog190** [johnnesky/beepbox#114]
current measure length is 128. The option to increase the song length currently is to change the beats per measure, but this messes up the simultaneous instruments per channel. Maybe increase to 256?

**Comment by M0ahgra** [comment]
i second this, once i had to make a very long song and i had to split it in half, really it doesnt make sense to have limits like this at all, just allow the user to choose.

**Comment by Mid-The-Modder** [comment]
Agreed. In the meantime, this is typically what modifications of the program are for, such as [JummBox](https://jummb.us) or [UltraBox](https://ultraabox.github.io), which increase the maximum amount of bars per song to 256 and 1024. You can import your BeepBox song via the URL or a .json file exported in the "file" menu and import it into one of these programs to continue extending your song.
